documentation/blog/2025-05-20-goose-gets-a-drivers-license/index.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Goose Gets a Driver's License!
+description: Control a MakeBlock mbot2 rover through MQTT and MCP as a Goose Extension
+authors: 
+    - ian
+---
+import YouTubeShortEmbed from '@site/src/components/YouTubeShortEmbed';
+
+![blog cover](goose-rover-blog.png)
+
+## I taught Goose how to drive (a rover)
+
+I had the pleasure of assembling a [MakeBlock mbot2 rover](https://www.makeblock.com/products/buy-mbot2) after seeing [a demo video](https://x.com/deemkeen/status/1906692248206524806) put together by [deemkeen](https://github.com/deemkeen) using MQTT and MCP. The mbot2 has a CyberPi module that runs MicroPython, and with a few instructions to connect to MQTT, it can be controlled by a Java-based MCP server, which allows Goose to control the rover!
+
+<!-- truncate -->
+
+<YouTubeShortEmbed videoUrl="https://www.youtube.com/embed/QKg2Q6YCzdw" />
+
+Deemkeen's instructions on getting started were pretty clear, and they did a great job of supplying some code to get started, but I wanted to take things a little further.

```

Comment:
If you use my suggestion above or something similar, you don't need this paragraph anymore. 

```suggestion

```